src/index.html

```diff
@@ -1,13 +1,369 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>Bose</title>
+    <link
+      rel="shortcut icon"
+      href="images/favicon.ico"
+      type="image/x-icon"
+    >
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700;800&family=Roboto:wght@400;700&display=swap" rel="stylesheet">
+
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page__body">
+    <header class="header page__section">
+      <div class="container">
+        <div class="header__content">
+          <div class="header__top">
+
+            <a href="/" class="logo">
+              <img
+                class="logo__image"
+                src="./images/icons/logo.png" alt="Bose">
+            </a>
+
+            <div class="header__top-icon">
+
+              <a href="tel:+1 234 555-55-55" class="icon icon--phone">
+                <div class="icon__tooltip">
+                  +1 234 555-55-55
+                </div>
+              </a>
+
```

Comment:
cannt see the icons may be change position of image or use translate to change position of the icons
<img width="666" alt="image" src="https://user-images.githubusercontent.com/100297341/191465504-a2db1038-10fc-4a5e-a9fe-a05c0fc4a699.png">

Reply:
I realized that the testimonials block didnt have display: grid property. I also added one more mixin for tablet size with another for loop. Seems crowded but I made it work :D